cvat/apps/engine/filters.py

```diff
@@ -306,33 +308,38 @@ def get_lookup_fields(self, view):
 
         return get_lookup_fields(view, fields=simple_filters)
 
-    def get_schema_fields(self, view):
-        assert coreapi is not None, 'coreapi must be installed to use `get_schema_fields()`'
-        assert coreschema is not None, 'coreschema must be installed to use `get_schema_fields()`'
-
-        lookup_fields = self.get_lookup_fields(view)
+    def get_schema_operation_parameters(self, view):
+        try:
+            queryset = view.get_queryset()
+        except Exception:
+            queryset = None
+            warnings.warn(
+                "{} is not compatible with schema generation".format(view.__class__)
+            )
```

Comment:
Are there any views left that are incompatible with schema generation? Why couldn't they be adapted?

Reply:
Probably, there are no such views and we can tighten this requirement.